### ISSUE TITLE
Issue with Jab (Thief skill)

### ISSUE THREAD

**Issue by dmhenderson76** [DarkCastleMUD/DarkCastle#114]
There's an issue where mobs that are immune to the jab skill are allowing players to circle them even when jab misses.  See below:

ack dodges the huge enfan's attack.
the huge enfan's claw cremates Jack dealing 70 damage!
Jack dodges the huge enfan's attack.
the huge enfan's claw cremates Jack dealing 75 damage!
Jack's pierce beats the shit out of the huge enfan delivering 142 damage!
Jack's pierce eviscerates the huge enfan inflicting 120 damage!

Z:163 R:30108 I:1> 
**Jack tried to jab the huge enfan in the head, but it dodged like a pro. <-- Jab miss**

Z:163 R:30108 I:1> 
Jack dodges the huge enfan's attack.
the huge enfan's claw cremates Jack inflicting 75 damage!
the huge enfan's claw obliterates Jack dealing 69 damage!
Jack dodges the huge enfan's attack.
the huge enfan deathstrokes Jack in her kidneys!  Jack cries out in pain!
Jack's pierce beats the shit out of the huge enfan causing 139 damage!
Jack's pierce eviscerates the huge enfan delivering 116 damage!

Z:163 R:30108 I:1> 
**Jack circles around her target... <--- Circles anyway???**
the huge enfan makes a strange gargling sound as Jack thrusts Janikhel's Blade of Shadows into its's back for 554 damage!
Jack's harm spell cuts into the huge enfan for 71 damage like the stroke of a broadsword!
the huge enfan makes a strange gargling sound as Jack thrusts A Huge Overtoned Phallus into its's back for 549 damage!

and immediately after that...

J:163 R:30108 I:1> 
**Jack tried to jab the huge enfan in the head, but it dodged like a pro. <--- Jab miss again (mob appears to be immune)**

Z:163 R:30108 I:1> 
Jack parries the huge enfan's attack.
the huge enfan's claw misses Jack.
Jack parries the huge enfan's attack.
the huge enfan's claw cremates Jack delivering 78 damage!
Jack's pierce beats the shit out of the huge enfan causing 134 damage!
Jack's pierce eviscerates the huge enfan dealing 123 damage!

Z:163 R:30108 I:1> 
Jack circles around her target...
**Jack thrusts Janikhel's Blade of Shadows into the huge enfan's back, felling it instantly with 661 damage. <---Circles anyway**
the huge enfan is DEAD!!
Your blood freezes as you hear the huge enfan's death cry.

**Comment by dmhenderson76** [comment]
Confirmed this is a bug - I can recreate it:


**6122/1722/-990 Target Is:   You try to jab the huge enfan, but fail. <---Jab miss**

6122/1718/-990 Target Is: (excellent condition) 
Your pierce eviscerates the huge enfan delivering 127 damage!
Your pierce beats the shit out of the huge enfan delivering 131 damage!
You dodge the huge enfan's attack.
You dodge the huge enfan's attack.
the huge enfan's claw cremates you causing 72 damage!
the huge enfan's claw obliterates you dealing 66 damage!

**5984/1718/-990 Target Is: (slightly hurt) You circle around your target...**
the huge enfan makes a strange gargling sound as you thrust Janikhel's Blade of Shadows in its back for 606 damage.
the huge enfan makes a strange gargling sound as you thrust the Master Assassin's Dagger in its back for 598 damage.

Interestingly, even though jab missed - I did get the "jab timer" thing so I couldn't just spam circle.  Jack was able to do it twice in a row though.
